apps/mobile-wallet/src/features/addressesManagement/AddressSettingsModal.tsx

```diff
@@ -53,8 +54,6 @@ const AddressSettingsModal = withModal<AddressSettingsModalProps>(({ id, address
 
     if (address.isDefault && !settings.isDefault) return
 
-    dispatch(activateAppLoading(t('Saving')))
```

Comment:
I noticed that after switching to the MMKV lib for storage, the storage operations are performed so fast that it actually creates a bug when showing and hiding the loading indication modal. I haven't figured out why the bug happens so I just removed the indication since the operation is instant.

Reply:
Oh woah. That's a good problem to have I guess 😆